xl-v1-object.go

```diff
@@ -429,6 +452,7 @@ func (xl xlObjects) PutObject(bucket string, object string, size int64, data io.
 		if vErr := data.(*signVerifyReader).Verify(); vErr != nil {
 			// Incoming payload wrong, delete the temporary object.
 			xl.deleteObject(minioMetaTmpBucket, tempObj)
+
 			// Error return.
```

Comment:
> in cache.create we do c.currentSize += uint64(size) while allocating new buffer. In that case we need not increment during cache.Create() and increment the cache size during Close() when adding to the cache. If the size exceeds cache size the []byte need not be added to the cache.

yes since Close() is used now.